testsuite/features/build_validation/add_custom_repositories/add_ceos8_repositories.feature

```diff
@@ -36,7 +45,8 @@ Feature: Adding the CentOS 8 distribution custom repositories
     When I follow the left menu "Software > Manage > Channels"
```

Comment:
add a scenario title before this line

something like `Scenario: Add both repositories to the channel`

Reply:
done